### PR TITLE
Fix verify and test-go

### PR DESCRIPTION
**Pull request by fejta**
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/test-infra/1076)

### CODE REVIEW

File: jenkins/job-configs/kubernetes-jenkins/bootstrap-ci-repo.yaml [hunk 1]
```diff
@@ -33,12 +33,11 @@
...
     repo-suffix:
-    # TODO(fejta): migrate critical jobs later
-    # kubernetes-verify-master:
-    #   branch: master
-    #   job-name: ci-kubernetes-verify-master
-    #   repo-name: k8s.io/kubernetes
-    #   frequency: 'H/5 * * * *'
+    - kubernetes-verify-master:
+        branch: master
+        job-name: ci-kubernetes-verify-master
```
Comment (ixdy) [inline jenkins/job-configs/kubernetes-jenkins/bootstrap-ci-repo.yaml]: has SQ been updated to look for a different job name?

**Comment by saad-ali** [comment]
We will need to repeat this song and dance once again for the `release-1.5` branch when we fast forward it and it also picks up PR https://github.com/kubernetes/kubernetes/pull/36834

**Comment by ixdy** [comment]
@saad-ali I think release-1.5 is already using the new bootstrap runner, so it should be fine.

**Comment by saad-ali** [comment]
> @saad-ali I think release-1.5 is already using the new bootstrap runner, so it should be fine.

I'm not very familiar with how it is setup. I see that https://github.com/kubernetes/test-infra/blob/master/jobs/ci-kubernetes-verify-release-1.5.sh#L35 calls `jenkins/gotest-dockerized.sh` so I assumed it would need to be similarly modified. If not, awesome!

**Comment by fejta** [comment]
Looking for an lgtm so I can merge and unblock things.

**Comment by saad-ali** [comment]
LGTM :)

**Pull request merged**